library/src/scripts/forms/checkRadioStyles.ts

```diff
@@ -224,6 +224,17 @@ export const checkRadioClasses = useThemeCache(() => {
         marginTop: unit(globalVars.spacer.size / 2),
     });
 
+    const grid = style("grid", {
```

Comment:
This will work when you have enough room and short enough text. I'm not overly concerned about the responsiveness for the dashboard, but you might want to make sure they stay aligned, you might want to opt for a css grid or an old school css table.

Reply:
You could probably still flex them, but just a flex basis is insufficient

![image](https://user-images.githubusercontent.com/1770056/78362834-c5a2bc00-7588-11ea-9b49-1ad37b322b0d.png)